### DISCUSSION TITLE
Issue with The Graph Query

### DISCUSSION THREAD

**Discussion by empeje** [f8n/fnd-protocol#89] [Q&A] [answered]
Hi Team,

I tried to use the Graph provided for the mainnet, but i have difficulty.

For example, if I want to query open buy nows for specific callers I tried the following

```graphql

{
 nftMarketBuyNows(orderBy: dateCreated, orderDirection: desc,
where: {seller:"0x5d7b71cda3ce450b5db872776c3524e574ee0b7a", status: Open}) {
    id,
    dateCreated,
    nft {
      id,
      tokenId,
      dateMinted,
    },
    nftContract {
      id,
      name,
      symbol
    },
    status,
    seller {
      id
    },
    amountInETH,
  }
}
```

the query above will return information, but if I change seller to `0xEF76173F4Eb704e83DBd4C51EDd147C95F3aDC7c` which is this profile https://foundation.app/@gorbymatheus I didn't get any buy nows however, in the foundation frontend there's a buy nows.

Thank you for your help.

**Comment by HardlyDifficult** [comment, marked as answer]
Subgraph queries are case sensitive - and for addresses Subgraph uses all lowercase while Foundation.app prefers the checksum format. So to run the query, you should first `.toLowerCase()` the address.  Then the query works with that other address, e.g.:

```graphql
{
 nftMarketBuyNows(orderBy: dateCreated, orderDirection: desc,
where: {seller:"0xef76173f4eb704e83dbd4c51edd147c95f3adc7c", status: Open}) {
    id,
    dateCreated,
    nft {
      id,
      tokenId,
      dateMinted,
    },
    nftContract {
      id,
      name,
      symbol
    },
    status,
    seller {
      id
    },
    amountInETH,
  }
}
```

hth - let us know if you have any other questions

**Reply by empeje** [reply]
Thanks @NickCuso it works like a charm :tada:
